docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -688,52 +975,58 @@ Ensure you have the following files in the current working directory.
-  |Kubernetes|Public Repo|`kubernetes`|`1.24.x`| `1.24.10` |
-  |Network|Public Repo|`cni-calico`|`3.25.x`|`3.25.0`|
-  |Storage|Public Repo|`csi-aws-ebs`|`1.16.x`|`1.16.0`|
-
-  Note that using this Terraform code will deploy the resources to AWS. To deploy your resource to Azure or Google Cloud, use the layer details outlined in [Cloud Service Provider Configurations] (https://github.com/spectrocloud/tutorials/tree/main/terraform/pack-tf/README.md#cloud-service-provider-configurations). 
+  |---|---|---|---|---|
+  |OS|Public Repo|`ubuntu-aws`|`LTS__22.4.x`| `22.04`|
+  |Kubernetes|Public Repo|`kubernetes`|`1.28.x`| `1.28.2` |
+  |Network|Public Repo|`cni-calico`|`3.26.x`|`3.26.1`|
+  |Storage|Public Repo|`csi-aws-ebs`|`1.22.x`|`1.22.0`|
+  |Authentication|Public Repo|`spectro-proxy`|`1.4.x`|`1.4.1`|
+  |App Services|Private Repo|`hellouniverse`|`1.0.x`|`1.0.0`|
 
 
 
-- **inputs.tf** - contains the variables used in the tutorial such as the names of cluster profile, cluster, cloud account, SSH key name, AWS region, pack name, and registry server. 
+- **inputs.tf** - contains the variables used in the tutorial, such as the names of cluster profile, cluster, cloud account, SSH key name, AWS region, pack name, and registry server. 
 
-  Some variables have a default value, but you *must* provide the values for `cluster_cloud_account_aws_name`, `aws_region_name`, `ssh_key_name`, and `private_pack_registry` variables. You will find a `#ToDo` tag next to each variable to update. Provide the values for these variables in a separate file, **terraform.tfvars**. Use default values for the remaining variables.
+  Some variables have a default value, but you *must* provide the values for the `cluster_cloud_account_aws_name`, `aws_region_name`, `ssh_key_name`, and `private_pack_registry` variables. You will find a `#ToDo` tag next to each variable that needs updating. Provide the values for these variables in a separate file, **terraform.tfvars**. Use the default values for the remaining variables.
 
 
 
-- **terraform.tfvars** - contains the variable definitions. The list of variables is outlined in the code block below. You *must* specify the values for all variables that are marked `"REPLACE ME"`. Read the inline comments below to understand each variable. 
+- **terraform.tfvars** - contains the variable definitions. The list of variables is outlined in the code block below. You *must* specify the values for all variables marked as `"REPLACE ME"`. Read the inline comments below to understand each variable. 
 
-  - For example, the value for  `cluster_cloud_account_aws_name` will be the name of the cloud account added to your Palette project settings. In this tutorial example, the cloud account name is **spectro-cloud**. 
+  - For example, the value for  `cluster_cloud_account_aws_name` should be the name of the cloud account added to your Palette project settings. In this tutorial, the cloud account name is **spectro-cloud**. 
```

Comment:
```suggestion
  - For example, the value for  `cluster_cloud_account_aws_name` should be the name of the cloud account added to your Palette project settings.
```

Let's not put our name as it can confuse the reader.